### PR TITLE
Updated ChatEntity and added ChatConfiguration

### PR DESCRIPTION
**Pull request by RamzesUAA**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by dreamofmaks — APPROVED** [review]

File: Car.Backend/Car.Domain/Services/Implementation/ChatService.cs [hunk 1]
```diff
@@ -19,7 +20,7 @@ public ChatService(IUnitOfWork<User> unitOfWorkUser, IUnitOfWork<Chat> unitOfWor
 
         public Chat GetChatById(int chatId)
         {
-            return unitOfWorkChat.GetRepository().Query(u => u.Users).FirstOrDefault(p => p.Id == chatId);
+            return unitOfWorkChat.GetRepository().Query(u => u.Reciver).FirstOrDefault(p => p.Id == chatId);
```
Comment (r-kovalch) [inline Car.Backend/Car.Domain/Services/Implementation/ChatService.cs]: Better to specify u in lambda explicitly

Comment (r-kovalch) [reply Car.Backend/Car.Domain/Services/Implementation/ChatService.cs]: also as p should be

File: Car.Backend/Car.Domain/Services/Implementation/ChatService.cs [hunk 2]
```diff
@@ -39,15 +40,15 @@ public Chat AddChat(Chat chat)
...
+            var chat = unitOfWorkChat.GetRepository().Query(u => u.Reciver).FirstOrDefault(c => c.Id == chatId);
+            chat.Reciver = user;
             unitOfWorkChat.SaveChanges();
             return user;
         }
 
         public List<Chat> GetUsersChats(int userId)
         {
-            var user = unitOfWorkUser.GetRepository().Query(u => u.Chats).FirstOrDefault(i => i.Id == userId);
+            var user = unitOfWorkUser.GetRepository().Query().Include(u => u.Chats).ThenInclude(c => c.Reciver).FirstOrDefault(i => i.Id == userId);
```
Comment (r-kovalch) [inline Car.Backend/Car.Domain/Services/Implementation/ChatService.cs]: Better to specify single char variables with full names

**Review by r-kovalch — COMMENTED** [review]
Minor readability improvements are needed

**Review by r-kovalch — APPROVED** [review]
Ought to amend the namings in lambdas

File: Car.Backend/Car.Data/Entities/Chat.cs [hunk 3]
```diff
@@ -8,6 +8,8 @@ public class Chat : IEntity
 
         public string Name { get; set; }
 
-        public ICollection<User> Users { get; set; } = new List<User>();
+        public User Reciver { get; set; }
```
Comment (NazariiChetvertukha) [inline Car.Backend/Car.Data/Entities/Chat.cs]: Please, rename to "Receiver"

**Pull request merged**